release/nightly_tests/dataset/map_batches_benchmark.py

```diff
@@ -0,0 +1,93 @@
+from typing import Literal
+
+import ray
+from ray.data.dataset import Dataset
+
+from benchmark import Benchmark
+
+
+def map_batches(
+    input_ds: Dataset,
+    batch_size: int,
+    batch_format: Literal["default", "pandas", "pyarrow", "numpy"],
+    num_calls: int,
+) -> Dataset:
+
+    ds = input_ds
+    for _ in range(num_calls):
+        ds = ds.map_batches(
+            lambda ds: ds, batch_format=batch_format, batch_size=batch_size
```

Comment:
The other dimension worth testing: task v.s. actor.

Reply:
@jianoaix - good call, added.